### DISCUSSION TITLE
Améliorer les filtres de recherche sur Admin

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1925] [Idées]
Suggestions pour améliorer le module de recherche sur la page "Cantines" du site Admin :
- avoir un filtre pour "public" / "privé"
- avoir des menus déroulants pour les filtres plutôt que la longue liste actuelle
- avoir les valeurs sélectionnées par catégorie de filtre qui s'affichent tout en haut du module de recherche

@JenniferStephan @hfroot @alemangui Est-ce facile à mettre en place?

**Comment by hfroot** [comment]
J'ai mis un filtre public/privé sur staging. J'ai aussi changé l'ordre des filtres pour mettre les filtres avec moins d'options en avance pour un peu resoudre le pb t'as évoqué. Mais j'aurai besoin de faire plus de recherche pour voir si les deux autres suggestions sont facile ou non

**Reply by ghost** [reply]
Merci déjà pour ces premières améliorations, @hfroot !
